docker-for-windows/edge-release-notes.md

```diff
@@ -12,6 +12,57 @@ For information about Stable releases, see the [Stable release
 notes](release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.2.3.0
+<Add date>
+
+> [Download](Add URL)
+
+### Upgrades
+
+- [Docker 19.03.8](https://github.com/docker/docker-ce/releases/tag/v19.03.8)
+- [Docker Compose 1.26.0-rc3](https://github.com/docker/compose/releases/tag/1.26.0-rc3)
+- [Linux 4.19.76](https://hub.docker.com/layers/docker/for-desktop-kernel/4.19.76-4e5d9e5f3bde0abf236f97e4a81b029ae0f5f6e7-amd64/images/sha256-11dc0f6ee3187088219ba1463ebb378f5093a7d98f176ddfd62dd6b741c2dd2d?context=repo)
+
+### New
+
+- Docker Desktop introduces a new onboarding tutorial upon first startup. The Quick Start tutorial guides users to get started with Docker in a few easy steps. It includes a simple exercise to build an example Docker image, run it as a container, push and save the image to Docker Hub.
+
+### Bug fixes and minor changes
+
```

Comment:
Should we maybe include a total 'count' of the number of public bugs we have fixed. Just to improve that 'why should I upgrade' moment for a user?

Reply:
Sure. Perhaps on the lines of 

`Docker Desktop Edge 2.2.3.0 fixes nn bugs reported on the docker/for-win and docker/for-mac GitHub repositories.`

@mat007 Would it be possible to get the number of bugs we've fixed on Edge, please? TIA

Reply:
Would summing up the number of links to `for-win` and `for-mac` tickets work for you @nebuk89 ?

Reply:
Yes exactly that is what customers would see as the 'number of fixes' and I think would be great!

Reply:
Or we could even just count the number of entries in «Bug fixes and minor changes», that’s 7 for the Mac and 28 for Windows!